content/docs/extensions/neon.md

```diff
@@ -4,11 +4,55 @@ enableTableOfContents: true
+The `neon` extension provides functions and views designed to gather Neon-specific metrics.  
 
-```bash shouldWrap
-psql 'postgresql://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/postgres?sslmode=require'
+## The `neon_stat_file_cache` view
 
+Neon computes have a local file cache that extends Postgres shared buffers to approximately 50% of your compute's RAM. The `neon_stat_file_cache` view provides the following data about local file cache usage:
+
+- `file_cache_misses`: The number of times requested data was not found in the local file cache.
+- `file_cache_hits`: The number of times requested data was found in the local file cache.
+- `file_cache_used`: The number of times the local file cache was used.
+- `file_cache_writes`: The number of writes to the local file cache.
+- `file_cache_hit_ratio`: The cache hit ratio for the local file cache. The file cache hit ratio is calculated according to the following formula:
+
+    ```
+    file_cache_hit_ratio = (file_cache_hits / (file_cache_hits + file_cache_misses)) * 100
+    ```
+
+    You can use the cache hit ratio as a measure of requests served from memory rather than disk. For better query performance, frequently accessed data should reside in memory. Generally, you should aim for a cache hit ratio of 99% or better.
+
+### Using the `neon_stat_file_cache` view
+
+To use the `neon_stat_file_cache` view, install the `neon` extension:
+
+```sql
+CREATE EXTENSION neon;
+```
+
+Issue the following query to view local file cache usage data for your compute instance:
+
+```sql
+SELECT * FROM neon.neon_stat_file_cache;
+ file_cache_misses | file_cache_hits | file_cache_used | file_cache_writes | file_cache_hit_ratio  
+-------------------+-----------------+-----------------+-------------------+----------------------
+           2133643 |       108999742 |             607 |          10767410 |                98.08
+(1 row)
+```
+
+<Admonition type="note">
+Statistics represent the lifetime of your compute, from the last time the compute started until the time you ran the query. Statistics are lost when your compute stops and gathered again from scratch when your compute restarts. Also, keep in mind that the statistics are for the entire compute, not specific databases or tables. Your compute runs an instance of Postgres, which may contain multiple databases and tables.
+</Admonition>
+
+For related information, see [How to size your compute](/docs/manage/endpoints#how-to-size-your-compute).
+
+## Views for Neon internal use
+
+The `neon` extension is installed to a system-owned `postgres` database in each Neon project by default. 
```

Comment:
I just noticed that I can access the "system-owned" postgres database as a user and have asked team compute if this is as intended here https://neondb.slack.com/archives/C04DGM6SMTM/p1709116034279269
This may be a bug.